Library/Homebrew/os/mac/xcode.rb

```diff
@@ -171,7 +173,8 @@ def detect_version_from_clang_version
         when 81      then "8.3"
         when 90      then "9.2"
         when 91      then "9.3"
-        else              "9.3"
+        when 100     then "10.0"
+        else              "10.0"
```

Comment:
what is the value for 9.4?

Reply:
I completely forgot 9.4 had been released. Should probably update the `when "10.13" then "9.3"` as well.

It's still 91/9.1 though:

```
Apple LLVM version 9.1.0 (clang-902.0.39.2)
Target: x86_64-apple-darwin17.7.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```